js/graph_config.js

```diff
@@ -269,13 +271,15 @@ GraphConfig.load = function(config) {
                 offset: 0,
                 power: 1.0,
                 inputRange: Math.max(Math.max(Math.abs(mm.max), Math.abs(mm.min)), 1.0),
-                outputRange: 1.0
+                outputRange: 1.0,
+				MinMax: mm,
+                EnabledMinMax: false
             };
         }
 
         const gyroScaleMargin = 1.20; // Give a 20% margin for gyro graphs
         const highResolutionScale = sysConfig.blackbox_high_resolution > 0 ? 10 : 1;
-
+        var mm = getMinMaxForFields(fieldName);
```

Comment:
Please do not use var. Try const or let instead.

Reply:
Ok. I will know.